components/automate-chef-io/content/docs/infra-server.md

```diff
@@ -0,0 +1,130 @@
++++
+title = "Install Chef Infra Server With Automate"
+description = "Use Chef Automate to install Chef Infra Server"
+date = 2020-02-11T14:24:00-08:00
+weight = 20
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "get_started"
+    weight = 60
++++
+
+Use Chef Automate to install Chef Infra Server either for a single-host installation that contains Chef Infra Server and Chef Automate, or for a standalone Chef Infra Server instance. See the [Chef Infra Server documentation](https://docs.chef.io/server_overview/) for
+instructions and guidance on using and managing your Chef Infra Server.
+
+## System requirements and prerequisites
+
+Before beginning your installation, check the [System Requirements]({{< relref "system-requirements.md" >}}) for Automate, and ensure that the `chef-automate` command line tool is installed.
+
+To download the `chef-automate` command line tool, run the following command in your command line interface:
+
+```shell
+curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
+```
+
+## Install Chef Automate and Chef Infra Server on the same host
+
+Use either a command line interface or a configuration file to install Chef Automate and Chef Infra Server on the same host. Run the commands as the superuser or use `sudo` at the start of each command.
+
+### Hardware Requirements
+
+For a single-host installation that contains Chef Infra Server and Chef Automate, we recommend the following memory and vCPU minimums:
+
+* up to 200 managed nodes: 8GB RAM, 2 vCPUs
+* between 200 and 500 managed nodes: 30GB RAM, 4 vCPUs
+* between 500 and 5000 managed nodes: 61GB RAM, 8vCPUs
+
+A single-host installation that contains Chef Infra Server and Chef Automate will require a `/hab` directory that contains 80 GB of disk space for software artifacts plus 2 MB of disk space per managed node.
+
+### Specify Products From the Command Line
```

Comment:
@yzl  Is there a distinction that needs to be made between "specify" in the header and "install" used in the content below? Asking to determine if wording should change through the rest of this section.

Reply:
@mjingle "Specify" is the mechanism. "Install" is the job to be done. We want to use "specify" in the header because the job to be done is always "install Automate", but users will chose different mechanisms based on their needs.

Reply:
Simple present, please. "...Chef Automate requires..."